db/migrate/20211118000056_create_jobs.rb

```diff
@@ -0,0 +1,14 @@
+class CreateJobs < ActiveRecord::Migration[6.0]
+  def change
+    create_table :jobs do |t|
+      t.string :title
+      t.string :location
+      t.text :description
+      t.date :date
+      t.time :time
+      t.integer :openings
```

Comment:
What is this for?

Reply:
I think he mentioned this to be how many people they are looking to fill the job